cfme/tests/control/test_actions.py

```diff
@@ -215,14 +215,16 @@ def action_collection(appliance):
     return appliance.collections.actions
 
 
-@pytest.fixture(scope="module")
+@pytest.yield_fixture(scope="module")
 def compliance_condition(appliance):
     condition_collection = appliance.collections.conditions
-    return condition_collection.create(
+    compliance_condition_ = condition_collection.create(
```

Comment:
REQUIRED: Please use leading underscore instead. pep8 states:
_single_leading_underscore: weak "internal use" indicator. E.g. from M import * does not import objects whose name starts with an underscore.
single_trailing_underscore_: used by convention to avoid conflicts with Python keyword, e.g.
Tkinter.Toplevel(master, class_='ClassName')